components/dashboard/src/admin/BlockedEmailDomains.tsx

```diff
@@ -95,37 +120,39 @@ export function BlockedEmailDomainsList(props: Props) {
                             <input
                                 className="w-64 pl-9 border-0"
                                 type="search"
-                                placeholder="Search by URL RegEx"
+                                placeholder="Search by domain"
                                 onChange={(v) => {
                                     setSearchTerm(v.target.value.trim());
                                 }}
                             />
                         </div>
                         <div className="flex space-x-2">
-                            <button onClick={add}>New Blocked Domain</button>
+                            <button onClick={add}>Add Domain</button>
                         </div>
                     </div>
                 </div>
 
-                <Alert type={"info"} closable={false} showIcon={true} className="flex rounded p-2 mb-2 w-full">
-                    Search by email domain URL using <abbr title="regular expression">RegEx</abbr>.
-                </Alert>
                 <div className="flex flex-col space-y-2">
                     <div className="px-6 py-3 flex justify-between text-sm text-gray-400 border-t border-b border-gray-200 dark:border-gray-800 mb-2">
-                        <div className="w-9/12">Domain URL (RegEx)</div>
+                        <div className="w-9/12">Domain</div>
                         <div className="w-1/12">Block Users</div>
```

Comment:
**question**: Not the best place to comment about this but should we enable this option when adding a new Blocked Email Domain? Same goes for the Blocked Repositories. Both can be considered out of the scope of this PR though. Your call. 🏓 

| BEFORE | AFTER |
|--------|--------|
| <img width="616" alt="Frame 1539" src="https://github.com/gitpod-io/gitpod/assets/120486/4f5e0751-0e41-46f0-b8dc-420161a51824"> | <img width="616" alt="Frame 1540" src="https://github.com/gitpod-io/gitpod/assets/120486/81894ac3-47c7-4c8b-be0b-36387368a49c"> |

Reply:
Yes, would make sense. But also not super important as this is only used by internal admins